src/components/number/number.test.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { createRef } from 'react';
+import {
+  render,
+  screen,
+  cleanup,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Number from './number.jsx';
+
+describe('Number', () => {
+  const requiredProps = {
+    id: 'test-component',
+    label: 'Test Component',
+  };
+
+  afterEach(cleanup);
```

Comment:
AFAIK, we don't actually need to cleanup anything. This is automatic in our environment.